spec.bs

```diff
@@ -439,9 +440,14 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
     1. [=Queue a global task=] on [=DOM manipulation task source=], given |global|, to resolve |p|
       with null.
   1. Otherwise:
-    1. [=Queue a global task=] on the [=DOM manipulation task source=], given |global|, to resolve
-      |p| with |winner|'s [=generated bid/ad descriptor=]. TODO: resolve |p| with urn-uuid, instead
-      of a URL.
+    1. Let |urn| and |fencedFrameConfig| be the result of TODO: constructing the relevant
+       {{FencedFrameConfig}} with |winner|'s [=generated bid/ad descriptor=].
+    1. Wait until |auctionConfig|'s [=auction config/resolve to config=] is a boolean.
+    1. Let |result| be |fencedFrameConfig|.
+    1. If |auctionConfig|'s [=auction config/resolve to config=] is false:
+      1. Set |result| to |urn|.
+    1. [=Queue a global task=] on the [=DOM manipulation task source=], given |global|, to
+       resolve |p| with |result|.

```

Comment:
This implies that the return type of `runAdAuction()` needs to be `Promise<(USVString or FencedFrameConfig)>`, right?

Reply:
Yes you're right, fixed.

Reply:
Oops, need to do it with the right <> chars.